src/components/LoanCards/Buttons/LoanBookmarkExp.vue

```diff
@@ -0,0 +1,100 @@
+<template>
```

Comment:
Was this file a complete duplicate of the existing version used in the KivaClassicLoanCard? If so, why?

Reply:
To have a more customizable component based on product requirements... do you suggest to upgrade the current to cover this new requirements?

Reply:
I do see the new design, just a little concerned about more duplicated query and logic code. Looking forward to a cleanup at some point.

Reply:
The hope is that this new card will be rolled out everywhere, so that's the plan...